package.json

```diff
@@ -23,6 +23,7 @@
   },
```

Comment:
Add `webdriverio` as a dev dep. Pro tip: try to avoid installing non-CLI packages globally on your host.